src/tracing/job.ts

```diff
@@ -203,6 +223,10 @@ async function traceWorkflowRunJob({
   const ctx = trace.setSpan(parentContext, parentSpan);
   const startTime = new Date(job.started_at);
   const completedTime = new Date(job.completed_at);
+  const annotationMessages = jobAnnotations
+    .map((a: JobAnnotation) => a.message)
+    .join(";");
```

Comment:
Joining several failure messages together isn't ideal, but currently don't see a better way to ensure we're capturing any/all failures (there can be more than one). This still gives us the ability to create an SLI based on matching specific keywords to this string.

Reply:
Unfortunately, Honeycomb doesn't have support for arrays. As for capturing these failure messages and possibly using these to created an SLI (based on a derived field), we can feature flag this using the [ci-experiments action](https://github.com/wrapbook/ci-experiments-action) to gather some data on a subset of the failures

Reply:
>we can feature flag this using the ci-experiments action to gather some data on a subset of the failures

We can do that!